galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -535,3 +535,15 @@ messages:
     description: "user namespace should be injectable if Istio is installed with enableNamespacesByDefault enabled and neither injection label is set."
     template: "is enabled for Istio injection, as Istio is installed with enableNamespacesByDefault as true."
     url: "https://istio.io/latest/docs/reference/config/analysis/ist0148/"
+
+  - name: "IstioProxyConfigMismatch"
+    code: IST0149
+    level: Warning
+    description: "The environment variable configured with the running proxies on pods does not match the environment variable defined in the mesh configmap."
+    template: "The environment variable configured with the running proxies on pods does not match the environment variable defined in the mesh configmap (dismatch environment variable in pod is %s and in mesh configmap is %s). This often happens when re-installing Istio control-plane and updating the mesh configmap."
```

Comment:
```suggestion
    template: "The environment variable %q configured with the running proxies on pods does not match the environment variable %q defined in the mesh configmap. This often happens when re-installing the Istio control plane and updating the mesh configmap."
```

Reply:
It's done, thanks @shamsher31!  And below is the output:

```
......
Warning [IST0149] (Pod httpbin-74fb669cc6-9tcrx.default) The environment variable "{\"tracing\":{\"zipkin\":{\"address\":\"jaeger-collector.istio-system.svc.cluster.local:9411\"}}}" configured with the running proxies on pods does not match the environment variable "{\"tracing\":{\"zipkin\":{\"address\":\"jaeger-collector.istio-system.svc.cluster.local:14268\"}}}" defined in the mesh configmap. This often happens when re-installing the Istio control plane and updating the mesh configmap.
Warning [IST0149] (Pod sleep-557747455f-xnjt9.default) The environment variable "{\"tracing\":{\"zipkin\":{\"address\":\"jaeger-collector.istio-system.svc.cluster.local:9411\"}}}" configured with the running proxies on pods does not match the environment variable "{\"tracing\":{\"zipkin\":{\"address\":\"jaeger-collector.istio-system.svc.cluster.local:14268\"}}}" defined in the mesh configmap. This often happens when re-installing the Istio control plane and updating the mesh configmap.
......
```